cmd/tracker/audit.go

```
@@ -29,22 +30,28 @@ func listRuns(workdir string) error {
 // printRunList prints the formatted run listing table.
 func printRunList(runs []tracker.RunSummary) {
 	fmt.Println()
-	fmt.Printf("  %-14s  %-8s  %-6s  %-8s  %-10s  %-26s  %s\n", "Run ID", "Status", "Nodes", "Retries", "Duration", "Bundle", "Failed At")
-	fmt.Printf("  %-14s  %-8s  %-6s  %-8s  %-10s  %-26s  %s\n", "──────", "──────", "─────", "───────", "────────", "──────", "─────────")
+	fmt.Printf("  %-14s  %-10s  %-6s  %-8s  %-10s  %-26s  %s\n", "Run ID", "Status", "Nodes", "Retries", "Duration", "Bundle", "Failed At")
+	fmt.Printf("  %-14s  %-10s  %-6s  %-8s  %-10s  %-26s  %s\n", "──────", "──────", "─────", "───────", "────────", "──────", "─────────")
 
 	for _, r := range runs {
 		icon := "+"
```

golangci-lint (2.12.2)

[error] ineffectual assignment to icon

(ineffassign)